docs/src/modules/components/MarkdownElement.js

```diff
@@ -267,18 +267,39 @@ const Root = styled('div')(
       },
     },
     '& .MuiCallout-root': {
+      display: 'flex',
+      gap: 12,
       padding: '16px',
       margin: '16px 0',
       border: '1px solid',
+      color: `var(--muidocs-palette-text-secondary, ${lightTheme.palette.text.secondary})`,
```

Comment:
<img width="673" alt="Screenshot 2023-08-18 at 16 51 23" src="https://github.com/mui/material-ui/assets/3165635/408c4cc8-d4bf-4127-bfea-97e8fddb9d34">

This feels a bit low in contrast. For a callout, we might expect the opposite: equal or more contrast, e.g. https://nextjs.org/docs/app/building-your-application/styling/css-in-js

<img width="301" alt="Screenshot 2023-08-18 at 16 51 40" src="https://github.com/mui/material-ui/assets/3165635/0c2a4795-c965-4139-a0e8-2939a9806385">

To consider

Reply:
I can always put it `text.primary`, but `secondary` seems to have enough contrast, too? Though, overall, not too concerned as these "naked" callouts shouldn't be used.